src/celengine/skygrid.cpp

```diff
@@ -569,9 +569,7 @@ SkyGrid::render(Renderer& renderer,
 
     double arcStep = (maxTheta - minTheta) / (double) ARC_SUBDIVISIONS;
     double theta0 = minTheta;
-
-    vector<LineStripEnd> buffer;
-    buffer.reserve(2 * (ARC_SUBDIVISIONS + 2));
+    static vector<LineStripEnd> buffer(2 * (ARC_SUBDIVISIONS + 2), { Vector3f::Identity(), 1.0f });
```

Comment:
what's the problem with the current code apart that it's slower?

Reply:
The current one uses .reserve and then assigns value like [0] [1] directly. This causes undefined behavior (crashes app built with MSVC debug)

Reply:
okay. clear. so it's better to use memory chunk allocated with `new`, but let's live with this